.github/actions/upgrade-provider/action.yml

```diff
@@ -0,0 +1,117 @@
+name: 'Upgrade provider'
+description: 'An action to upgrade bridged providers'
+inputs:
+  kind:
+    description: The kind of upgrade to perform; valid options are ["all", "provider", "bridge"]
+    required: true
+    default: "all"
+  username:
+    description: 'The username for git configuration'
+    required: false
+    default: ${{ github.actor }}
+  email:
+    description: 'The email for git configuration'
+    required: false
+  automerge:
+    description: 'If the created PR should be auto-merged'
+    required: false
+    default: false
+  pr-assign:
+    description: |
+      A user to assign the upgrade PR to
+    required: true
+    default: ${{ github.actor }}
```

Comment:
Might be easier to upstream this change if you make `pr-assign` optional and set it to `${{ github.actor }}` in the workflow files instead